tests/UnifiedSpecTests/UnifiedTestRunnerTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace MongoDB\Tests\UnifiedSpecTests;
+
+use MongoDB\Tests\FunctionalTestCase;
+
+class UnifiedTestRunnerTest extends FunctionalTestCase
+{
+    public function testEntityMapObserver(): void
+    {
+        $test = UnifiedTestCase::fromFile(__DIR__ . '/crud/find.json');
```

Comment:
For the sake of completeness, this test might start failing if the crud tests change significantly. Creating a separate test would be safer, but for the sake of getting this method tested (and thus used) this suffices.

Reply:
Indeed, I'll create a file specifically test the tester.

Reply:
File added.